# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1760028130 -7200
# Parent  609e318d1601c985b5b0056ff2f68318f4f22855
Misc: Show folder path in message list header tooltip.

The hunk in folder-tree-row.mjs is from bug 382977, https://phabricator.services.mozilla.com/D268146

diff --git a/mail/base/content/about3Pane.js b/mail/base/content/about3Pane.js
--- a/mail/base/content/about3Pane.js
+++ b/mail/base/content/about3Pane.js
@@ -4819,9 +4819,9 @@ var threadPaneHeader = {
       return;
     }
 
     this.folderName.textContent = gFolder?.abbreviatedName ?? document.title;
-    this.folderName.title = gFolder?.prettyName ?? document.title;
+    this.folderName.title = gFolder?.prettyPath ?? document.title;
     this.updateMessageCount(
       gFolder?.getTotalMessages(false) || gDBView?.numMsgsInView || 0
     );
     this.updateSelectedCount();
diff --git a/mail/base/content/widgets/folder-tree-row.mjs b/mail/base/content/widgets/folder-tree-row.mjs
--- a/mail/base/content/widgets/folder-tree-row.mjs
+++ b/mail/base/content/widgets/folder-tree-row.mjs
@@ -303,9 +303,9 @@ class FolderTreeRow extends HTMLLIElemen
     this.setFolderPropertiesFromFolder(folder);
     this._nameStyle = nameStyle;
     this._serverName = folder.server.prettyName;
     this._folderName = folder.abbreviatedName;
-    this._fullFolderName = folder.name;
+    this._fullFolderName = folder.prettyPath;
     this._setName();
     const isCollapsed = this.classList.contains("collapsed");
     this.unreadCount = folder.getNumUnread(isCollapsed);
     this.totalCount = folder.getTotalMessages(isCollapsed);
